src/Field.php

```diff
@@ -337,11 +337,15 @@ public function normalize($value)
                 } elseif (is_string($value)) {
                     $value = new $class($value);
                 } elseif (!$value instanceof $class) {
-                    if (is_object($value)) {
-                        throw new ValidationException(['must be a '.$f->type, 'class' => $class, 'value class' => get_class($value)]);
+                    if ($value instanceof \DateTimeInterface) {
+                        $value = new $class($value->format('Y-m-d H:i:s.u'), $value->getTimeZone());
```

Comment:
should be `getTimezone` not `getTimeZone`.
https://www.php.net/manual/en/datetime.gettimezone.php

Reply:
Good catch, I will fix it promtly. I have copied this clever code from https://github.com/atk4/data/blob/37d1722ac47f53179677e9509f5e484fd3f8ed03/src/Persistence/SQL.php#L555 , so I will fix it there too.